docs/recent_activity/recent_activity.md

```diff
@@ -0,0 +1,66 @@
+---
+description: Monitor recent activity logged actions.
+edition: experience
+---
+
+# Recent activity log
+
+[[= product_name =]] logs various operations on the repository and in the application.
+
+If you have **Setup / Administrate** and **Activity Log / Read** [permissions](permission_system.md),
+you can review the most recent activity log in  the Back Office, **Admin** -> **Activity list**.
+
+![Activity list](4.6_activity_list.png "Activity list")
+
+By default, actions on the following items are displayed:
+
+- [Content](content_items.md)
+- [Location](manage_locations_urls.md#content-locations)
+- [Product](products.md)
+- [Product variant](work_with_product_variants.md)
+- [Site](work_with_sites.md)
+
+!!! note
+
+    If your implementation requires that other actions are logged,
+    see [custom log entry developer documentation]([[= developer_doc =]]/administration/recent_activity/recent_activity/#adding-custom-activity-log-entries).
+
+    By default, log entries are kept for 30 days.
+    This time can be modified through configuration.
+    For more information, see [developer documentation]([[= developer_doc =]]/administration/recent_activity/recent_activity/#configuration-and-cronjob).
+
+Log entries are grouped by date,
+then by logical bond (like web request, or migration file).
+
+Each activity log entry shows
+
+- when the action was performed,
+- who performed it (avatar, first name, last name),
+- the action itself as a verb,
+- and the item the action was performed on.
+
+!!! note
```

Comment:
It looks as if this part is doubled - see the note at line 23

Reply:
Duplicate removed in dba3bab2f699d21b2c84e77e3e8ed80ab43753d5